src/api-service/tests/test_task_config.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import pytest
+
+from __app__.onefuzzlib.tasks.config import is_valid_blob_name
+
+
+BLOB_NAME_TEST_CASES = [
```

Comment:
Looking at the spec that you included:

Avoid blob names that end with a dot (.), a forward slash (/), or a sequence or combination of the two. No path segments should end with a dot (.).

Looks like is_valid_blob_name("abcd.")    will return true

Reply:
Good idea, will add.